app/components/school_search_component/school_search_component.html.erb

```diff
@@ -1,22 +1,22 @@
 <%= tag.div id: id, class: class_names(classes, 'mt-4') do %>
-  <ul class="nav nav-tabs locales" id="search-tabs" role="tablist">
+  <ul class="nav nav-tabs url-aware locales" id="search-tabs" role="tablist">
```

Comment:
The url-aware class allows tabs to open when url contains anchor/id for tab. This was useful when linking from restricted prompt to schools page, to automatically open the school group tab. I did end up renaming the ids for the tabs to be #school rather than #school-content etc as this looked better in the url.